internal/db/schema/migrations/oss/postgres/19/01_db.up.sql

```diff
@@ -0,0 +1,56 @@
+begin;
```

Comment:
Should this be in a migration for everyone? Or should it be part of the tests utilizing it?

Reply:
I agree, that is shouldn't be.  With that said, this PR is aligned with the current pattern used by internal/db for test schema objects.  I've opened an issue to refactor all of them: https://hashicorp.atlassian.net/browse/ICU-2841 in a separate PR.